app/code/Magento/Sales/Test/Unit/Model/OrderTest.php

```diff
@@ -403,9 +403,6 @@ public function testGetCustomerName(array $expectedData)
         $this->order->setCustomerFirstname($expectedData['first_name']);
         $this->order->setCustomerSuffix($expectedData['customer_suffix']);
         $this->order->setCustomerPrefix($expectedData['customer_prefix']);
-        $this->scopeConfigMock->expects($this->exactly($expectedData['invocation']))
```

Comment:
Could you please add case to cover "middle" name too? It seems `getCustomerMiddlename()` may return `null` 
https://github.com/magento/magento2/blob/9f43821af62ab177e44b51b112ff8d32d9b82813/app/code/Magento/Sales/Model/Order.php#L2044

Reply:
Hi @xmav , I've added changes to the test, please check.

please also check this change [setup/src/Magento/Setup/Model/BatchInsert.php](https://github.com/magento/magento2/pull/34689/files#diff-7c879d7e7310d11559221d4be3be7349d8f1933c8f30c315a87249d7623e586fR88)